### PR TITLE
iptables_linux.go: use kmp rather than bm

### PR DESCRIPTION
**Pull request by bassosimone**
This is a workaround for #10

### CODE REVIEW

File: iptables/iptables_linux.go [hunk 1]
```diff
@@ -19,15 +19,15 @@ func (s *linuxShell) rstIfDestinationEqualsAndIsTCP(ip string) error {
 
 func (s *linuxShell) dropIfContainsKeyword(keyword string) error {
 	return shellx.Run(
-		"iptables", "-A", "OUTPUT", "-m", "string", "--algo", "bm",
+		"iptables", "-A", "OUTPUT", "-m", "string", "--algo", "kmp",
```
Comment (xhdix) [inline iptables/iptables_linux.go:22]:
```suggestion
		"iptables", "-I", "OUTPUT", "-m", "string", "--algo", "kmp",
```

File: iptables/iptables_linux.go [hunk 2]
```diff
@@ -19,15 +19,15 @@ func (s *linuxShell) rstIfDestinationEqualsAndIsTCP(ip string) error {
...
 		"--string", keyword, "-j", "DROP",
 	)
 }
 
 func (s *linuxShell) rstIfContainsKeywordAndIsTCP(keyword string) error {
 	return shellx.Run(
 		"iptables", "-A", "OUTPUT", "-m", "string", "--proto", "tcp", "--algo",
```
Comment (xhdix) [inline iptables/iptables_linux.go:29]:
```suggestion
		"iptables", "-I", "OUTPUT", "-m", "string", "--proto", "tcp", "--algo",
```

**Review by xhdix — DISMISSED** [review]
According to : https://github.com/ooni/jafar/issues/10#issuecomment-564527822
(Lines 10 and 15 should also be changed to `I`.)

**Comment by xhdix** [comment]
(Lines `10` and `15` should also be changed to `I`.)

**Comment by bassosimone** [comment]
We agreed offline with @xhdix that the correct fix is to jump to a chain as described in #1

**Pull request merged**